src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -410,7 +413,29 @@ def create(self, validated_data: dict):
                 }
             )
 
-        return super().create(validated_data)
+        obj = super().create(validated_data)
+        track_object_serializer(obj, self)
```

Comment:
Is it correct that the dict inside this function is the reason why you had to rewrite `__hash__` function?

Reply:
correct - django's base `__hash__` requires `self.pk` to not be `None`. In the CMIS case we can rely on the `uuid` being unique and act like a PK for that purpose, even though there's no DB record in postgres.

Reply:
Thanks